lib/vm/src/export.rs

```diff
@@ -103,21 +103,21 @@ impl VMExportTable {
         self.from.style()
     }
 
-    /// Returns whether or not the two `VMExportTable`s refer to the same Memory.
+    /// Returns whether or not the two `VMTable`s refer to the same Memory.
     pub fn same(&self, other: &Self) -> bool {
         Arc::ptr_eq(&self.from, &other.from)
     }
 }
 
-impl From<VMExportTable> for VMExport {
-    fn from(table: VMExportTable) -> Self {
+impl From<VMTable> for VMExtern {
+    fn from(table: VMTable) -> Self {
         Self::Table(table)
     }
 }
 
 /// A memory export value.
-#[derive(Debug, Clone)]
-pub struct VMExportMemory {
+#[derive(Debug, Clone, MemoryUsage)]
+pub struct VMMemory {
```

Comment:
No action required. FWIW I think that naming these things VM* is a bug. It's not a new bug introduced in your PR. VM* should mean "used by the VM" as in "the generated code uses this" which implies "must be repr(C) and contain things that are available in repr(C)" which would exclude Arc for instance (I've been arguing we should make a VMArc).